drivers/can/can_mcp2515.c

```diff
@@ -468,13 +588,20 @@ static void mcp2515_handle_interrupts(struct device *dev)
 			mcp2515_tx_done(dev, 2);
 		}
 
-		/* clear the flags we handled */
-		mcp2515_cmd_bit_modify(dev, MCP2515_ADDR_CANINTF, canintf,
-				       ~canintf);
+		if (canintf != 0) {
+			/* Clear TX flags manually */
```

Comment:
@alexanderwachter added the the error interrupt flag as well, can either drop the comment as it is fairly self explanatory or just say `clear remaining flags`.

Reply:
Done